h/static/scripts/root-thread.js

```diff
@@ -65,6 +65,8 @@ function RootThread($rootScope, annotationUI, features, searchFilter, viewFilter
           return thread.annotation && metadata.isAnnotation(thread.annotation);
         } else if (state.selectedTab === uiConstants.TAB_NOTES) {
           return thread.annotation && metadata.isPageNote(thread.annotation);
+        } else {
+          return true;
```

Comment:
That's true. Since our current tools cannot prove that this branch is unreachable, a better option might be to throw an error.